### ISSUE TITLE
Swiper buttons are clickable despite of button-disabled class is attached to it/ the left and right arrows are clickable for the search similar items under the product

### ISSUE THREAD

**Issue by swap12875** [topcoder-archive/TCO21-Regionals-QA-Competition#187]
### Description :  
Swiper buttons are clickable despite of button-disabled class is attached to it/ the left and right arrows are clickable for the search similar items under the product

### Steps to Reproduce:
1. Login to the "https://secure.newegg.com/"
2. from the left side product category menu, select any  of the product category
3. preferably select the following 
 * Home >> Computer Systems>> Gaming Desktops >> ABS 
 4. now select any product from the list
 5. On the product info page, scroll till the 'Similar item' section
 6. click on the left or right arrow continuously , observe the class attached to the buttons on click events.
 7. the button despite of having Button-disabled class attached to it, are clickable. the same left or right button can be used for scrolling the similar product from left to right and vice-e versa.

### Snap(s) / Video(s):

https://user-images.githubusercontent.com/67191980/128257553-b48d7c99-61a2-455a-92b1-37fbdaf7d0ff.mp4

### Actual Result(s):
*  despite of having Button-disabled class attached to it, are clickable. the same left or right button can be used for scrolling the similar product from left to right and vice-e versa.

### Expected Result(s):
* The button should be disable when there is no item present towards left or right, whichever is the applicable case.

### Environment Details:
* **Device(s)**: Lenovo Ideapad i5 14-Inch
* **Resolution**: 1366 X 768
* **Operating System**: Windows 10 Pro (64 Bit)
* **Browser(s)**: Chrome [Version 92.0.4515.131 (Official Build) (64-bit)]

**Comment by Rokit12** [comment]
Invalid
https://monosnap.com/direct/IhiZjOdVO6ruLMetJz8qNHEdPnxyTw

**Comment by swap12875** [comment]
Fine @Rokit12, request you to please follow the video shared in the bug report, as per the repro step -6 , you should be clicking on either the left or right arrow (only )continuously, but not the left and right arrow simultaneously (as clicked be you, in your video). your clarification comments will be more clear if you justify the logic you want to convey if it is 'Invalid' ( as per the challenge details, you should put the  comment to support your challenge. this is the procedure)

![image](https://user-images.githubusercontent.com/67191980/128283744-ea56b2c8-5882-4589-afce-bacc17165273.png)

**Comment by Rokit12** [comment]
Yes, valid despite the button disabled as it reaches the end,they are still swipped from one swipe button
https://monosnap.com/direct/FzudXu3PcDMx28JNQUqjYywEEZuGYN

**Comment by swap12875** [comment]
@Rokit12, is it correct if you removed the 'not a bug' label here? as you have changed to a valid issue

**Comment by codejamtc** [comment]
**Submitter**: 8 Points
**Challenger**: 1 Point
